knowledge_base/save_job_result_to_volume/resources/my_volume.volume.yml

```diff
@@ -0,0 +1,9 @@
+resources: 
+  volumes: 
+    my_volume:
+      catalog_name: main
+      name: my_volume
+      # We use the ${resources.schemas...} interpolation syntax to force the creation
+      # of the schema before the volume. Usage of the ${resources.schemas...} syntax 
+      # allows Databricks Asset Bundles to form a dependency graph between resources.
+      schema_name: ${resources.schemas.trips.name}
```

Comment:
Can you order them how they are used, i.e. catalog, then schema, then name.

Then it reads like how you would address it.

Reply:
done